modules/test/playwright/pages/commerce/commerceAdminProductDetailsProductRelationsPage.ts

```diff
@@ -5,23 +5,69 @@
 
 import {Locator, Page} from '@playwright/test';
 
+import {CommerceAdminProductDetailsPage} from './commerceAdminProductDetailsPage';
 import {CommerceDNDTablePage} from './commerceDNDTablePage';
 
 export class CommerceAdminProductDetailsProductRelationsPage extends CommerceDNDTablePage {
+	readonly addProductRelationHeading: (
+		productName: string
+	) => Promise<Locator>;
+	readonly CommerceAdminProductDetailsPage: CommerceAdminProductDetailsPage;
```

Comment:
CommerceAdminProductDetailsPage -> commerceAdminProductDetailsPage

Reply:
Cannot be non-capital letter it has to match the class name.